Products/PleiadesEntity/content/Named.py

```diff
@@ -119,6 +120,30 @@ def getLocations(self):
                  or checkPermission('Pleiades: View link to draft', o))
         ]
 
+    security.declareProtected(permissions.View, 'getLocations')
+    def getSubConnections(self):
+        """
+        """
+        checkPermission = getSecurityManager().checkPermission
+        return [
+            o for o in self.values()
+            if interfaces.IConnection.providedBy(o)
+            and (checkPermission(permissions.View, o)
+                 or checkPermission('Pleiades: View link to draft', o))
+        ]
+
+    security.declareProtected(permissions.View, 'getLocations')
+    def getReverseConnections(self):
+        """
+        """
+        checkPermission = getSecurityManager().checkPermission
+        return [
+            o for o in self.getBRefs('connection')
+            if interfaces.IConnection.providedBy(o)
+            and (checkPermission(permissions.View, o)
+                 or checkPermission('Pleiades: View link to draft', o))
+        ]
```

Comment:
I debated that, and thought we might eventually want to make the Feature connections (which are the other Named super-class) use this same connection mechanism. Since the very similar getLocations and getNames methods live here, I decided to keep it all in one place.